activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+*   Fix accidental building of child records when `inverse_of` is specified
+
+    If an existing record with inverse_of children was duplicated,
+    additional child records would be built. These new records could
+    be persisted upon save or update inadvertently.
+
+    This adds a method to check that new records with inversing have an
+    existing owner:
+
+    ```ruby
+    new_record_inverse_of_existing_owner?(record, inversing: false)
+    ```
```

Comment:
I don't think we need this in the changelog, as it's an implementation detail, and not something users will interact with.